docs/sources/configuration/flags.md

```diff
@@ -43,6 +43,14 @@ These statistics are sent to `stats.grafana.org`.
 Statistics help us better understand how Grafana Agent is used.
 This helps us prioritize features and documentation.
 
+Besides the enabled feature flags we are also collecting the following information:
+* A randomly generated and an anonymous unique ID (UUID).
+* Timestamp of when the UID was first generated.
+* Timestamp of when the report was created (by default every 4h).
+* Version of running Grafana Agent.
+* Operating system Grafana Agent is running on.
+* System architecture Grafana Agent is running on.
```

Comment:
IMO I'd change the first paragraph to not mention the feature flags:

```
By default, Grafana Agent sends anonymous, but uniquely-identifiable usage information for your running Grafana Agent instance to Grafana Labs.
These statistics are sent to `stats.grafana.org`.
``` 

And then change this to be a fully exhaustive list: 

```
The usage information includes the following details:
* A randomly generated and an anonymous unique ID (UUID).
* Timestamp of when the UID was first generated.
* Timestamp of when the report was created (by default every 4h).
* Version of running Grafana Agent.
* Operating system Grafana Agent is running on.
* System architecture Grafana Agent is running on.
* List of enabled feature flags. 
```

I would also add a disclaimer that this list may change over time, and if they do, we will describe changes in the changelog.